### PR TITLE
Upgrade cypress to latest

### PR DESCRIPTION
**Pull request by sava-savov-ontotext**

### CODE REVIEW

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

File: test-cypress/support/index.js [hunk 1]
```diff
@@ -19,10 +19,6 @@ import './commands';
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
 
-// Configures retry count for failed tests TODO: Remove after tests are stabilized
-require('cypress-plugin-retries');
-Cypress.env('RETRIES', 0);
-
 // Configures an environment variable with the key used for common actions (cmd on mac, ctrl on other OS).
 // This variable must be used in all actions that type e.g. ctrl-a to select text.
 Cypress.env('modifierKey', Cypress.platform === 'darwin' ? '{cmd}' : '{ctrl}');
```
Comment (desislava-hristova-ontotext) [inline test-cypress/support/index.js:24]: Is it possible to use the platform type instead of a sever name?

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Review by desislava-hristova-ontotext — APPROVED** [review]

**Pull request merged**